tardis/montecarlo/base.py

```diff
@@ -106,7 +107,7 @@ def legacy_update_spectrum(self, no_of_virtual_packets):
             self.spectrum_virtual.update_luminosity(
                 self.montecarlo_virtual_luminosity)
 
-    def run(self, model, no_of_packets, no_of_virtual_packets=0, nthreads=1):
+    def run(self, model, no_of_packets, no_of_virtual_packets=0, nthreads=1,last_run=False):
```

Comment:
I think we could get away without introducing this new argument.
What we should do (if possible), is to move the postprocess to `MontecarloRunner.calculate_Edotlu(self, model)`.
This method could then be called anytime. It would make sense for Simulation to call this function after It's done with it's last iteration.